src/text_render.rs

```diff
@@ -311,6 +472,11 @@ impl TextRenderer {
         viewport: &Viewport,
         text_areas: impl IntoIterator<Item = TextArea<'a>>,
         cache: &mut SwashCache,
+        #[cfg(feature = "custom-glyphs")] rasterize_custom_glyph: impl FnMut(
```

Comment:
I guess this could also be a `Result` with an `Err` when custom glyph rasterization fails. We could bubble this up as a new variant of `PrepareError` too so you could know when the custom rasterizer failed (instead of the glyph being skipped)

Reply:
Makes sense. I'll work on this.